epistles-v2/original/02-epistles.adoc

```diff
@@ -724,8 +735,14 @@ So no more but my love.
 [.signed-section-signature]
 G+++.+++ F.
 
+[.postscript]
+====
+
 Read this in your men and women`'s meeting.
 
+====
+
+[.signed-section-context-close]
 Maryland, the 24th of the 12th month, 1672.
```

Comment:
I wasn't sure if you wanted to keep this below the postscript? There's one other instance of it, and then a couple in chapter 3.

Reply:
I think it's good this way 👍